pkg/v1/providers/tests/unit/tkg-bom-v1.4.0.yaml

```diff
@@ -0,0 +1,383 @@
+apiVersion: run.tanzu.vmware.com/v1alpha2
```

Comment:
nit: I think this may be better placed in `pkg/v1/providers/tests/unit/fixtures`